ui/components/password-reset-form.js

```diff
@@ -2,19 +2,17 @@ import { useRouter } from 'next/router'
 import { useState } from 'react'
 import { useSWRConfig } from 'swr'
 
-export default function PasswordResetForm() {
+export default function PasswordResetForm({ header, subheader }) {
   const { mutate } = useSWRConfig()
   const router = useRouter()
   const { token } = router.query
 
   const [password, setPassword] = useState('')
   const [error, setError] = useState('')
-  const [submitting, setSubmitting] = useState(false)
 
   async function onSubmit(e) {
     e.preventDefault()
-
-    setSubmitting(true)
```

Comment:
I think this was to stop double clicking on the "reset password button" and submitting twice, it doesnt look like this behavior is accounted for anymore?